packages/profile/src/views/Profile.jsx

```diff
@@ -207,6 +256,64 @@ const Profile = () => {
               <Input key={fieldName} fieldName={fieldName} state={state} dispatch={dispatch} />
             )
           })}
+          <div className='mt-8 mb-2 w-min'>
+            <label className='label block mb-1'>
+              <span className='label-text text-gray-700 capitalize'>Avatar</span>
+            </label>
+
+            <div className='mb-2 text-sm text-gray-700'>
+              Upload a JPEG or PNG. Recommended size is 500 x 500.
```

Comment:
why jpeg and png?

Reply:
Those are just the ones I know should render properly for everyone. What would you suggest?